frontend/public/style/_overrides.scss

```diff
@@ -188,6 +188,12 @@ form.pf-v5-c-form {
   --pf-v5-c-data-list__item--BackgroundColor: transparent;
 }
 
+// ask sgoodwin for advice
```

Comment:
@sg00dwin, this case is specific to a dropdown group.  https://github.com/openshift/console/pull/13406 doesn't cover it.  What's a better way to fix this case?

Reply:
`<DropdownGroupDeprecated>` adds a plain `<ul>` so unfortunately it will require this specific targeted css rule, such as you have, to remove the bullets and padding.

The other options are to use the current PF `Dropdown` component or not use `DropdownGroupDeprecated` and instead apply the html markup as below which will be covered by the existing css rules to remove the bullets and padding.

```
<section class="pf-v5-c-dropdown__group">
  <h1 class="pf-v5-c-dropdown__group-title">
    {t('public~Log actions')}
  </h1>
  <ul className="pf-v5-c-dropdown__menu-list">
    <DropdownItemDeprecated ...
  </ul>
</section>
```

Reply:
Switching to the PF5 `Dropdown` means this fix is no longer needed.